internal/discoverer/discoverer.go

```diff
@@ -0,0 +1,87 @@
+package discoverer
+
+import (
+	"errors"
+	"fmt"
+	"github.com/api7/apisix-seed/internal/utils"
```

Comment:
Please configure your editor to solve the imports.

Reply:
Done.